examples/vue/src/app/pages/Home.vue

```diff
@@ -30,7 +20,8 @@ export default defineComponent({
     const count = ref(0)
 
     const onConfirm: onConfirm = (value: string) => {
-      alert(value)
+      console.log('onConfirm', value)
```

Comment:
can you please remove this console log?

Reply:
this is an example app, so it is okay :-)